src/utils.cpp

```diff
@@ -139,7 +139,7 @@ void StdioSilencer::silence()
     close(new_stdio);
   } catch (const std::system_error &e) {
     if (errno == EMFILE)
-      LOG(FATAL) << e.what() << ": please raise NOFILE";
+      LOG(BUG) << e.what() << ": please raise NOFILE";
```

Comment:
Shouldn't use `throw FatalUserException`? The original code suggests that we consider `EMFILE` a user-caused error while everything else an internal bpftrace error.

Reply:
I wasn't sure about this one. I'll change it.